### PR TITLE
Feature/ab#539 create template with postgres

### PR DESCRIPTION
**Pull request by witchpou**

### CODE REVIEW

File: generator-templates/postgres/application/properties.ftl [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+spring.jpa.hibernate.naming.physical-strategy=de.${app.packageName?lower_case}.persistence.config.DatabasePhysicalNamingStrategy
+spring.datasource.username=${app.baseName?lower_case}
+spring.datasource.password=${app.baseName?lower_case}
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
+#spring.jpa.hibernate.ddl-auto=create
+
+# Flyway
+spring.flyway.user=${r"${spring.datasource.username}"}
+spring.flyway.password=${r"${spring.datasource.password}"}
+spring.flyway.url=${r"${spring.datasource.url}"}
```
Comment (DerCed) [inline generator-templates/postgres/application/properties.ftl:28]: What does this r do?

File: generator-templates/postgres/deployment/localenv-docker-compose.ftl [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  postgres:
+    container_name: ${app.baseName?lower_case}-db
+    image: postgres:latest
+    environment:
+      POSTGRES_DB: ${app.baseName?lower_case}
+      POSTGRES_USER: ${app.baseName?lower_case}
+      POSTGRES_PASSWORD: ${app.baseName?lower_case}
+      PGDATA: /var/lib/postgresql/data
+    healthcheck:
+      test: ['CMD-SHELL', 'pg_isready -U ${app.baseName?lower_case}'] # <<<---
```
Comment (DerCed) [inline generator-templates/postgres/deployment/localenv-docker-compose.ftl:12]: Comment?

File: generator-templates/postgres/deployment/localenv-docker-compose.ftl [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      KEYCLOAK_IMPORT: /opt/jboss/keycloak/imports/realm.json
+      DB_VENDOR: postgres
+      DB_ADDR: ${app.baseName?lower_case}-db-keycloak
+      DB_PORT: 5432
+      DB_USER: 'keycloak'
+      DB_PASSWORD: 'keycloak'
+      PROXY_ADDRESS_FORWARDING: 'true'
+      KEYCLOAK_FRONTEND_URL: 'http://localhost:8080/auth'
+    ports:
+      # <Port exposed> : < MySQL Port running inside container>
```
Comment (DerCed) [inline generator-templates/postgres/deployment/localenv-docker-compose.ftl:76]: Comment

File: generator-templates/postgres/deployment/postgreslocal-docker-compose.ftl [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  postgres:
+    container_name: ${app.baseName?lower_case}-db
+    image: postgres:latest
+    environment:
+      POSTGRES_DB: ${app.baseName?lower_case}
+      POSTGRES_USER: ${app.baseName?lower_case}
+      POSTGRES_PASSWORD: ${app.baseName?lower_case}
+      PGDATA: /var/lib/postgresql/data
+    healthcheck:
+      test: ['CMD-SHELL', 'pg_isready -U ${app.baseName?lower_case}'] # <<<---
```
Comment (DerCed) [inline generator-templates/postgres/deployment/postgreslocal-docker-compose.ftl:12]: See above

File: generator-templates/postgres/flyway/V1_0__init-postgres.ftl [hunk 5]
```diff
@@ -0,0 +1,95 @@
+<#list app.entities as entity>
+CREATE SEQUENCE IF NOT EXISTS "${entity.name?lower_case}_id_seq";
+
+CREATE TABLE "${entity.name?lower_case}"
+(
+<#if entity.fields??>
+    <#list entity.fields as field>
+    <#if field.fieldType == "String" || field.fieldType == "Enum">
+    "${field.fieldName?lower_case}" VARCHAR(255)<#if field.required> NOT NULL </#if>,
+    <#--  <#if field.fieldValidateRulesMaxlength??>, length=${field.fieldValidateRulesMaxlength}</#if>)  -->
```
Comment (DerCed) [inline generator-templates/postgres/flyway/V1_0__init-postgres.ftl:10]: Comment

**Review by DerCed — APPROVED** [review]
I approve this PR, but there are some minor things, that could be fixed

File: generator-templates/postgres/application/properties.ftl (hunk 1 above)
Comment (witchpou) [reply generator-templates/postgres/application/properties.ftl:28]: ${r" is used to print out the content without replacing stuff inside ${

**Pull request merged**